cmd/osbuild-composer/composer.go

```diff
@@ -197,9 +201,25 @@ func (c *Composer) Start() error {
 		}()
 	}
 
-	if c.kojiListener != nil {
+	if c.apiListener != nil {
 		go func() {
-			err := c.koji.Serve(c.kojiListener)
+			const apiRoute = "/api/composer/v1"
```

Comment:
ideally the version would come from the openapi.yml spec, but if there isn't an elegant way to do it, it's no big deal

Reply:
I had them in there at first (in `api.go`), but did not want to depend on those packages in the main binary. I think it should not, because the top-level entry points should really all be specified next to each other to ensure they're not conflicting, no?

Reply:
that is a good point, agreed